twarep/electrifly
# Simulation

## Checklist
**Note:** Put an x into the box like this [x] to check these off

## What I Did
Edited these files:
![image](https://github.com/twarep/electrifly/assets/64234627/12e2c2d6-1f47-4ffe-b17d-e2effe2f2e82)


## How I Did it?
With logic.

## How to Test?
run forecast.py and simulation.py. If it doesn't work run all the files we changed and then run simulation.py. You can check the outputs and do sanity checks with the times and outputs

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
